### PR TITLE
Reenable flannel on arm64

### PR DESCRIPTION
**Pull request by glevand**
Also adds missing ebuild metadata.

### CODE REVIEW

File: app-admin/flannel-wrapper/flannel-wrapper-0.7.0-r2.ebuild [hunk 1]
```diff
@@ -23,11 +23,24 @@ RDEPEND="
...
+		tag="v${PV}-${ARCH}"
+	fi
+
 	exeinto /usr/lib/coreos
 	doexe "${FILESDIR}"/flannel-wrapper
 
-	systemd_dounit "${FILESDIR}"/flanneld.service
-	systemd_dounit "${FILESDIR}"/flannel-docker-opts.service
+	sed "s|@FLANNEL_IMAGE_TAG@|${tag}|g" \
+		"${FILESDIR}"/flanneld.service > flanneld.service
```
Comment (crawford) [inline app-admin/flannel-wrapper/flannel-wrapper-0.7.0-r2.ebuild]: We typically use the temp directory for these intermediate artifacts.

File: app-admin/flannel-wrapper/flannel-wrapper-0.7.0-r2.ebuild [hunk 2]
```diff
@@ -23,11 +23,24 @@ RDEPEND="
...
+	cat flanneld.service
```
Comment (crawford) [inline app-admin/flannel-wrapper/flannel-wrapper-0.7.0-r2.ebuild]: Leftover debug?

**Comment by glevand** [comment]
Put temp files in temp directory.  Removed forgotten debug code.

**Comment by glevand** [comment]
Rebased to latest, simplified 'tag' logic in ebuild.

**Comment by glevand** [comment]
Rebased to latest.

@crawford The changes you reqested have been made, so this should be ready to merge.

**Pull request merged**